functions.php

```diff
@@ -61,4 +62,38 @@ function eachCharacter(array $character): string
         $characterHtmlElementsString .= $characterHtmlElement;
     }
     return $characterHtmlElementsString;
+}
+
+
+
+/**
+ * this extracts the TV shows' names
+ *
+ * @param PDO $db this extracts from the database and returns in an associative array
+ *
+ * @return array an array of the TV show names
+ *
+ */
+
+function extract_tvShow_from_db(PDO $db): array
+{
+    $query = $db->prepare('SELECT `id`, `name` FROM `TVshows` ORDER BY `name` ASC;');
+    $query->execute();
+    return $query->fetchAll();
+}
+
+
+/**
+ * This adds an array of given values of a new character to the database
+ *
+ * @param array $newInputArray this is the array of the character's values being added to the db
+ *
+ * @param PDO $db this is the database that the values will be added to
+ *
+ */
+
+function addNewItem_to_db (array $newInputArray, PDO $db) {
+    $query = $db->prepare('INSERT INTO `cartoons`(`character_name`, `TVshow_id`, `IQ`, `image`) 
+VALUES (:characterName, :tvShowName, :characterIq, :characterImgLink);');
+    $query->execute($newInputArray);
```

Comment:
execute returns a boolean, true if it works, false if it fails

Reply:
OK, sorry I just don't understand what I should do instead. What do you suggest?